### ISSUE TITLE
Item.centerpoint() NPE

### ISSUE THREAD

**Issue by sethrem** [powerbot/powerbot#1227]
Sometimes for some reason ctx.inventory.select().id(882).poll().centerPoint() throws a NPE
    0231 java.lang.NullPointerException 
        org.powerbot.script.rt4.Item.centerPoint(hc:250)
         ge.poll(ge.java:116)
	 org.powerbot.script.PollingScript$6.run(ok:129)

    ctx.input.click(ctx.inventory.select().id(882).poll().centerPoint(), true)

**Comment by Timer** [comment]
```
java.lang.NullPointerException 
org.powerbot.script.rt4.Item.centerPoint(Item.java:55)
ge.poll(ge.java:116)
org.powerbot.script.PollingScript$6.run(PollingScript.java:74)
```

**Comment by Timer** [comment]
This was resolved in f5deee8c9fae32538bcde6c9331eae0aae2ae2f5. Please wait for next release.
